ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx

```diff
@@ -49,7 +49,8 @@ const BancomatDetailScreen = ({ route }: Props) => {
         abiCode: O.some(abiInfo?.abi),
         holderName: O.fromNullable(nameSurname),
         expireMonth: O.fromNullable(info.expireMonth),
-        expireYear: O.fromNullable(info.expireYear)
+        expireYear: O.fromNullable(info.expireYear),
+        bankName: O.some(abiInfo?.name)
```

Comment:
since the bank name has an handled null case later on in the code, I don't think it's a good idea to possibly make the whole `Sequence` fail because of a missing a11y label, which by itself is non-breaking